lib/loader/file_loader.rb

```diff
@@ -39,7 +39,10 @@ def load(filename, filehandle: Zinzout.zin(filename), skip_header_match: nil)
       skip_matching_header(filehandle, skip_header_match).each.lazy
         .map { |line| log_and_parse(line) }
         .chunk_while { |item1, item2| item1.batch_with?(item2) }
-        .each { |batch| batch_loader.load(batch) }
+        .each do |batch|
+          batch_loader.load(batch)
+          Thread.pass
```

Comment:
Sidekiq complains if your jobs run for too long without passing. I'm not convinced they need to pass after every batch, and the threading in general might be a source of the bad performance I was seeing in dev (i.e. it might be better just to start a bunch of processor containers, rather than a single one with many threads)

Reply:
See DEV-405